tests/thread/thread.edl

```diff
@@ -40,6 +40,25 @@ enclave {
             [out] size_t* max_readers,
             [out] size_t* max_writers,
             [out] bool* readers_and_writers);
+
+        public void enc_spin_lock(
+            [user_check] void * lock);
```

Comment:
Instead of user check, you could use and `out` parameter which is a better practice that we want to encourage our users to adopt.

Reply:
ok, thanks